api/app/tests/hfi/test_hfi_endpoint_request.feature

```diff
@@ -62,3 +62,19 @@ Feature: /hfi/
             | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/planning_area/1/station/230/fuel_type/2                   | None                | 200         | hfi/test_hfi_endpoint_response_set_fuel_type.json        | True          | test_hfi_endpoint_stored_request.json |
             # Invalid fuel type should return 500 error, and not be saved.
             | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/planning_area/1/station/230/fuel_type/-1                  | None                | 500         | None                                                     | False         | None                                  |
+
+    Scenario: HFI - Admin POST add station
+        Given I received a POST request for hfi-calc admin <url> with <role>
```

Comment:
if we're being REALLY pedantic about grammar, I think it should be "Given I _send_ a POST request ..."

Reply:
I'd rather not these tests are brittle...